demo/hyproof-demo.md

```diff
@@ -141,5 +146,25 @@ The truth of this data could then be confirmed against the cryptographic data wi
 
 In this way `Heidi` and `Emma` keep each other honest, and the `Regulator` keeps them both honest.
 
+### 4. Revoking the certificate
+
+One final action that is possible within the system is the ability for `Reginald the Regulator` to revoke a certificate.
+
+`Reginald` has identified a problem with this latest hydrogen certificate.
+
+Click on the `Revoke` button on the right hand side of the screen. 
+
+In order to keep `Reginald` themselves honest and auditable, the reasons for revocation must be recorded in an indelible, high integrity manner, and made available to others. This form asks `Reginald` to supply the reasons for revocation.
+
+![Revocation Form](./revocation_form.png)
+
+`Reginald` clicks submit, and the form is loaded into the system and attached to the relevant certificate.
+
+After a few seconds of processing the certificate is now `Revoked`. This is instantly visible to all.
+
+![Revoked Certificate](./revoked_certificate.png)
```

Comment:
This is more like some sort of FYI comment .. I think I am supposed to remove the (OPEN ATTACHMENT) a link in the next PR but we can def keep the image as is as it is not that far away from the actual final thing.

Reply:
I will update the docs once that's done